src/main/java/kr/sesac/aoao/server/diary/controller/DiaryController.java

```diff
@@ -0,0 +1,98 @@
+package kr.sesac.aoao.server.diary.controller;
+
+import java.net.URI;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import kr.sesac.aoao.server.diary.controller.dto.request.DiaryCreateRequest;
+import kr.sesac.aoao.server.diary.controller.dto.request.DiaryUpdateRequest;
+import kr.sesac.aoao.server.diary.controller.dto.response.GetDiaryDetailResponse;
+import kr.sesac.aoao.server.diary.controller.dto.response.GetDiaryResponse;
+import kr.sesac.aoao.server.diary.service.DiaryService;
+import kr.sesac.aoao.server.global.controller.dto.response.ApplicationResponse;
+import lombok.RequiredArgsConstructor;
+
+/**
+ * @since 2024.01.23
+ * @author 최정윤
+ */
+@RestController
+@RequiredArgsConstructor
+public class DiaryController<DiaryUpdateRequest> {
```

Comment:
```suggestion
@RestController
@RequestMappring("/diray")
@RequiredArgsConstructor
public class DiaryController {
```

`@RequestMapping` 을 사용하면 중복적인 URI 를 제거할 수 있습니당 ㅎㅎㅎ 그리고 <클래스 타입> 은 현재 컨트롤러에서는 필요하지 않은것 같은데 제거하면 어떨까요!!

Reply:
감사합니다! 덕분에 더 코드가 깔끔해졌네요 ㅎㅎ